.travis.yml

```diff
@@ -1,20 +1,23 @@
 language: cpp
-compiler:
-  - gcc
-  - clang
-os:
-  - linux
-  - osx
 dist: bionic
-osx_image: xcode11.5
-matrix:
-  exclude:
+jobs:
+  include:
     - os: osx
+      osx_image: xcode11.5
+      compiler: clang
+      env: SUBTARGET=tiny   MAME=mametiny64
+    - os: linux
+      arch: amd64
       compiler: gcc
+      env: SUBTARGET=tiny   MAME=mametiny64
     - os: linux
-      compiler: clang
-env:
-  - SUBTARGET=tiny   MAME=mametiny64
+      arch: arm64
+      compiler: gcc
+      env: SUBTARGET=tiny   MAME=mametiny
```

Comment:
Are you telling it to build a 32-bit ARM binary on an AArch64 host here, or does it not add the `64` suffix for AArch64, or am I missing something?

Reply:
The latter - there is no 64 suffix on aarch64. This is set in row 366 of the make makefile or thereabouts. I vaguely recall this is due to the fact that non-empty architecture results in -m32 or -m64 being passed to GCC which is not supported on arm.